### PR TITLE
CVSL-1911: In vary case load handler, handle null release dates by di…

### PR DESCRIPTION
**Pull request by justin-gardiner**
…splaying release date as 'Not Found'.

### CODE REVIEW

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/routes/varyingLicences/handlers/caseload.ts [hunk 1]
```diff
@@ -20,9 +20,10 @@ export default class CaseloadRoutes {
         ? await this.comCaseloadService.getTeamVaryCaseload(user, req.session.teamSelection)
         : await this.comCaseloadService.getStaffVaryCaseload(user)
     ).map(comCase => {
+      const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'Not Found'
```
Comment (SebNegahbanMoJ) [inline server/routes/varyingLicences/handlers/caseload.ts:23]: *nitpick: * Elsewhere in the app we use lowercase of 'not found'
If this ends up being a short-term fix, I think it's fine to go out, but if it hangs around more than a couple of days, UCD may want us to correct it

**Review by SebNegahbanMoJ — APPROVED** [review]

File: server/routes/varyingLicences/handlers/caseload.ts (hunk 1 above)
Comment (justin-gardiner) [reply server/routes/varyingLicences/handlers/caseload.ts:23]: ok I may as well fix it now

Comment (justin-gardiner) [reply server/routes/varyingLicences/handlers/caseload.ts]: Fixed, can you approve when it's built again, thanks

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**